docs/registry/attributes/workflow.md

```diff
@@ -0,0 +1,126 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Workflow
+
+## Workflow Attributes
+
+Workflows seek to describe the execution of one ore more tasks to achieve an end result.
+Workflows can be used to describe cicd, deployments, generic jobs or even cron jobs.
+
+| Attribute | Type | Description | Examples | Stability |
+|---|---|---|---|---|
+| <a id="workflow-definition-id" href="#workflow-definition-id">`workflow.definition.id`</a> | string | Unique ID for this workflow which can be used to correlate multiple executions of this workflow. | `5c6e9a4b-69ab-499f-b4a7-4cf5c8720d66` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-definition-name" href="#workflow-definition-name">`workflow.definition.name`</a> | string | Name of the workflow | `Build and deploy application` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-definition-type" href="#workflow-definition-type">`workflow.definition.type`</a> | string | Type of the workflow | `DSL` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-definition-url-full" href="#workflow-definition-url-full">`workflow.definition.url.full`</a> | string | The [URL](https://wikipedia.org/wiki/URL) of the workflow, providing the complete address in order to locate and identify the workflow. | `https://github.com/open-telemetry/semantic-conventions/actions/runs/9753949763?pr=1075` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-definition-version" href="#workflow-definition-version">`workflow.definition.version`</a> | string | Version of the workflow | `2.1` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-execution-attempt" href="#workflow-execution-attempt">`workflow.execution.attempt`</a> | int | What number execution attempt is this. [1] | `1` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="workflow-execution-id" href="#workflow-execution-id">`workflow.execution.id`</a> | string | Unique ID for this execution of the workflow and will be shared across all the task runs. | `5c6e9a4b-69ab-499f-b4a7-4cf5c8720d66` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
I understand that there is good reason to have `workflow.execution.id` and `task.run.id` as a separate concept, but would there be some way of mating this with trace and span ids? I feel like the `workflow.execution.id` should be used as trace id and each `task.run.id` should show up as span id somewhere too.